src/main/java/wolox/training/models/Book.java

```diff
@@ -24,6 +25,7 @@
     @SequenceGenerator(name = "BOOK_SEQ", sequenceName ="BOOK_SEQ")
     private long id;
     @Column
+    @ApiModelProperty(notes ="The book genre: could be horror, comedy,drama,etc.")
     private String genre;
     @NotNull
     @Column(nullable = false)
```

Comment:
you should document all or most of the atributes

Reply:
I made documentations to all atributes